app/models/work_packages/scopes/relatable.rb

```diff
@@ -40,33 +40,114 @@ module Relatable
       # * Ancestor/descendant: relations cannot be drawn between ancestor and descendants. It is important to note though,
       #   that relations can be created between any two work packages within the same tree as long as they are not in a direct
       #   or transitive ancestor/descendant relationship. So a relation between siblings is just as possible as one between
-      #   aunt and nephew.
+      #   aunt and nephew. Additionally, a transitive relationship which leaves the tree for any one hop, is exempt from that rule
+      #   so it is possible to have child --- blocks ---> other --- blocks ---> parent.
```

Comment:
> And I'm not sure what a transitive ancestor/descendant relationship is exactly.

My understanding of a transitive relationship as opposed to a direct relationship is that a transitive relationship consists of 2 or more direct relationships. So the work packages involved might not be directly connected but are in effect still connected. In the example above, 47-"parent" would be transitively related to 50-"child child" since they are both connected to 48-"child". For ancestors/descendants that distinction admittedly might not be as important as it is for other relations like follows/precedes.

> But it's possible to set 48 as a parent, despite having a transitive ancestor/descendant relationship (if I understood what transitive means correctly). Is it intended or is it a bug?

This is a valid relation since for the 'relates' type, some limitations do not apply (As you have noted yourself in a different comment)

Reply:
Then this whole paragraph is wrong:

> * Ancestor/descendant: relations cannot be drawn between ancestor and descendants. It is important to note though, that relations can be created between any two work packages within the same tree as long as they are not in a direct or transitive ancestor/descendant relationship. So a relation between siblings is just as possible as one between aunt and nephew. Additionally, a transitive relationship which leaves the tree for any one hop, is exempt from that rule so it is possible to have child --- blocks ---> other --- blocks ---> parent.

The first sentence: "relations cannot be drawn between ancestor and descendants.", and yet a "relates-to" is ok... This should be mentioned here.

It could even be shorter: 

* Ancestor/descendant: relations can be created between any two work packages within the same tree as long as they are not in a ancestor/descendant relationship.
  For instance, a relation between siblings is possible, a relation between aunt and nephew is possible, whereas a relation between a parent and a child is not possible, and a relation between grand parent and grand child is not possible either.
  The only exception is for the "relates" relation which can apply to ancestors / descendants, but can not apply to direct parent / child because of the previous rule.

Reply:
I read up on your suggestion a bit late so I already changed the paragraph recently and did not include your suggestion. Can you please take a look to see if that new text is better already.

The exception is still not excluded since there is an extra paragraph just for the exceptions and additions to the rules which I think makes it easier to understand the rules first, and then see to the special cases. Especially if the additional rules for the PARENT relationship were to be included as well, it would be way harder to understand.